css/navbar.css

```diff
@@ -0,0 +1,66 @@
+* {
```

Comment:
Common css can be moved to separate file

Reply:
Okay

Reply:
Thanks